spec/features/consumer/cookies_spec.rb

```diff
@@ -52,6 +60,14 @@
   end
 
   describe "policy page" do
+
+    # keeps config unchanged
+    around do |example|
+      original_config_value = Spree::Config[:cookies_policy_matomo_section]
```

Comment:
I see you've added this for these 2 specs, but is there a global config to disable it on all the other specs by default?

Reply:
Ah ok, I wasn't sure about the defaults.